test/extended/util/workloads.go

```diff
@@ -25,46 +25,46 @@ type DeployOption func(*Deployment)
 // SetDeploymentName sets the deployment's name
 func SetDeploymentName(name string) DeployOption {
 	return func(dep *Deployment) {
-		dep.name = name
+		dep.Name = name
 	}
 }
 
 // SetDeploymentNameSpace sets the deployment's namespace
 func SetDeploymentNameSpace(namespace string) DeployOption {
 	return func(dep *Deployment) {
-		dep.namespace = namespace
+		dep.Namespace = namespace
 	}
 }
 
 // SetDeploymentReplicas sets the deployment's replicas
 func SetDeploymentReplicas(replicas string) DeployOption {
 	return func(dep *Deployment) {
-		dep.replicas = replicas
+		dep.Replicas = replicas
 	}
 }
 
 // SetDeploymentAppLabel sets the deployment's appLabel
 func SetDeploymentAppLabel(appLabel string) DeployOption {
 	return func(dep *Deployment) {
-		dep.appLabel = appLabel
+		dep.AppLabel = appLabel
 	}
 }
 
 // SetDeploymentImage sets the deployment's image
 func SetDeploymentImage(image string) DeployOption {
 	return func(dep *Deployment) {
-		dep.image = image
+		dep.Image = image
 	}
 }
 
 // NewDeployment creates a new customized deployment object
 func NewDeployment(opts ...DeployOption) Deployment {
 	defaultDeployment := Deployment{
-		name:      "e2e-dep-" + GetRandomString(),
-		namespace: "default",
-		replicas:  "1",
-		appLabel:  "e2e-app-" + GetRandomString(),
-		image:     "gcr.io/kuar-demo/kuard-amd64:blue",
+		Name:      "e2e-dep-" + GetRandomString(),
+		Namespace: "default",
```

Comment:
@Phaow in our automation can we change the namespace to be something else other than default ? I think dev might be doing the same in their e2e tests.

Reply:
We could do it, need another step to create ns.

Reply:
Thanks, @xingxingxia Can you as well check if you are okay with this change ?

Reply:
@kasturinarra @Phaow I'm OK with this change. If wanting other else, we can add one more function like `func CreateSetDeploymentNameSpace(namespace string) DeployOption {`

Reply:
Sure, thanks!

Reply:
Addressed.